### ISSUE TITLE
Create API endpoints for several different raster operations

### ISSUE THREAD

**Issue by kellyi** [azavea/geotrellis-collections-api-research#5]
TODO: update this card with more specifics once #3 is complete.

**Comment by kellyi** [comment]
| Path | Service |
| --- | --- |
| [/localvariety](http://localhost:7000/localvariety) | Returns local variety results |
| [/focalstandarddeviation](http://localhost:7000/focalstandarddeviation) | Returns focal standard deviation results |
| [/zonalhistogram](http://localhost:7000/zonalhistogram) | Returns zonal histogram results |
| [/pngtile](http://localhost:7000/pngtile) | Returns a PNG tile fit to the shape |
| [/geotiff](http://localhost:7000/geotiff) | Returns a GeoTIFF for the drawn shape along with Raster data |

**Comment by kellyi** [comment]
Server's currently set up to create an S3LayerReader from a catalog. However, it is not currently querying for any specific RDDs from the catalog, and the endpoints don't currently do anything with the data.

Some steps here:
- [x] move the ops out into a separate jobUtils file like the one from mmw-geoprocessing
- [x] set up localvariety endpoint to get counts from one of the RDDs
- [x] ~figure out the focal and zonal ops~ replaced this with some count and average operations

**Comment by kellyi** [comment]
Going to create a new issue for the png/geotiff tile operations:

> set up visual tile endpoints such that one uses nlcd and the other uses slope to return a tile cut to the extent of the posted polygon
